pyoptsparse/pySLSQP/pySLSQP.py

```diff
@@ -166,7 +167,10 @@ def __call__(
             # SLSQP - Objective/Constraint Values Function
             # =================================================================
             def slfunc(m, me, la, n, f, g, x):
-                fobj, fcon, fail = self._masterFunc(x, ["fobj", "fcon"])
+                if (x < blx).any() or (x > bux).any():
+                    warnings.warn("Values in x were outside bounds during a "
```

Comment:
This line [broke the build](https://dev.azure.com/mdolab/Public/_build/results?buildId=8201&view=logs&j=2d879094-0a86-54e9-c117-83eecb3ade7b&t=28b2e3e5-a4a9-54fc-8c3d-6c70517ff4c0&l=93), would you mind fixing it?
Also, could you use the [pyOptSparseWarning class](https://github.com/mdolab/pyoptsparse/blob/main/pyoptsparse/pyOpt_error.py) instead to clarify that this warning comes from pyOptSparse?

Reply:
Done! But just a note, there may be some cases where those might get a little verbose as it could get raised quite a few times compares to the python warning class that I believe shouldn't re-trigger

![image](https://github.com/user-attachments/assets/3b52a914-b5cc-4283-8519-0a5fae231a8e)